src/dex/balancer-v3/balancer-v3-pool.ts

```diff
@@ -197,19 +198,16 @@ export class BalancerV3EventPool extends StatefulEventSubscriber<PoolStateMap> {
       return null;
```

Comment:
as we update states for all pools, not only for the one ones, I guess we should not return here